build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/__component_version__/test/test-amp-__component_name_hyphenated__.js

```diff
@@ -35,6 +35,7 @@ describes.realWin(
     });
 
     it('example test renders', async () => {
+      // __do_not_submit__: This is example code only.
```

Comment:
This test is fine to keep in a submitted extension, no?
```suggestion
```

Reply:
Not the most useful but I guess it doesn't hurt